aws/gpu_tree_shap/gpu_tree_shap.ipynb

```diff
@@ -0,0 +1,493 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        # using default "tree_method": "gpu_hist" and "predictor": "gpu_predictor"
</pre><p>Move these comments to the script or above, this can be confusing as they might try to just uncomment this line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>